### PR TITLE
Publish IAS crate

### PR DESCRIPTION
**Pull request by jethrogb**

### CODE REVIEW

File: intel-sgx/ias/examples/attestation-tool.rs [hunk 1]
```diff
@@ -0,0 +1,204 @@
+/* Copyright (c) Fortanix, Inc.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#[macro_use]
+extern crate clap;
```
Comment (dingxiangfei2009) [inline intel-sgx/ias/examples/attestation-tool.rs:8]: Since we are using Rust Edition 2018, we may just go with
```suggestion
use clap::clap_app;
```

**Review by janosimas — DISMISSED** [review]

File: intel-sgx/ias/src/api.rs [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+
+use sgx_isa::{Attributes, Miscselect, Report};
+use std::str;
+use std::fmt;
+use byteorder::{BigEndian, ReadBytesExt};
+use serde::{Serialize, Deserialize};
+
+#[derive(Debug, Clone, Copy, Ord, PartialOrd, Eq, PartialEq, Hash)]
+pub enum IasVersion {
+    V2 = 2,
```
[the remark this answers is not in the document]
Comment (jethrogb) [reply intel-sgx/ias/src/api.rs:15]: This is correct. Are you suggesting a change?

[the remark this answers is not in the document]
Comment (janosimas) [reply intel-sgx/ias/src/api.rs:15]: It's a non-blocker suggestion. The default is version 4, so I don't think it's an blocker for publishing this first version.

**Comment by Pagten** [comment]
Looks like this crate's tests haven't been added to Travis CI yet.

**Comment by Pagten** [comment]
Looks like this crate's tests haven't been added to Travis CI yet.

**Comment by Taowyoo** [comment]
I think this PR is already done by https://github.com/fortanix/rust-sgx/pull/406
And ias crate is already published see https://crates.io/crates/ias/.
So I will close it. Please be free to open it again if I am wrong.

**Pull request closed without merging**